Dashboard Pod crashes after deployment of StressChaos/TimeChaos Experiments

## Bug Report

**What version of Kubernetes are you using?**
AWS EKS 1.18

**What did you do?**
I installed Chaos-Mesh using the[ helm Chart ]( https://github.com/chaos-mesh/chaos-mesh/tree/master/helm/chaos-mesh 
) for the installation. After successful installation of the ChaosMesh everything is working fine when deploying e.g. PodChaos Experiments. 
As soon as i deploy either TimeChaos or StressChaos Experiments the Dashboard Pod crashes.

```
2020-10-23T07:57:55.747Z        DEBUG   controller-runtime.controller   Successfully Reconciled {"controller": "timechaos", "request": "confluentplatform/time-shift-example"}
2020-10-23T07:57:55.767Z        DEBUG   controller-runtime.controller   Successfully Reconciled {"controller": "timechaos", "request": "confluentplatform/time-shift-example"}
2020-10-23T07:58:05.021Z        DEBUG   controller-runtime.controller   Successfully Reconciled {"controller": "timechaos", "request": "confluentplatform/time-shift-example"}
panic: reflect: call of reflect.Value.IsZero on zero Value

goroutine 627 [running]:
reflect.Value.IsZero(0x0, 0x0, 0x0, 0x1bde45c)
        /usr/local/go/src/reflect/value.go:1118 +0x4b6
github.com/chaos-mesh/chaos-mesh/api/v1alpha1.(*TimeChaos).GetChaos(0xc0020db340, 0xc0020db340)
        /src/api/v1alpha1/zz_generated.chaosmesh.go:925 +0x204
github.com/chaos-mesh/chaos-mesh/api/v1alpha1.(*TimeChaosList).ListChaos(0xc0004369a0, 0x2113940, 0xc002084900, 0x20dc880)
        /src/api/v1alpha1/zz_generated.chaosmesh.go:955 +0x149
github.com/chaos-mesh/chaos-mesh/pkg/apiserver/experiment.(*Service).state.func1(0x445736, 0x1eef340)
        /src/pkg/apiserver/experiment/experiment.go:959 +0x118
golang.org/x/sync/errgroup.(*Group).Go.func1(0xc001805f20, 0xc0020849c0)
        /root/go/pkg/mod/golang.org/x/sync@v0.0.0-20190911185100-cd5d95a43a6e/errgroup/errgroup.go:57 +0x59
created by golang.org/x/sync/errgroup.(*Group).Go
        /root/go/pkg/mod/golang.org/x/sync@v0.0.0-20190911185100-cd5d95a43a6e/errgroup/errgroup.go:54 +0x66
```
The same problem occurs wether i deploy it trough the dashboard itself or through kubetl apply with a yaml file. 
I even tried it with the [example from the Documentation](https://chaos-mesh.org/docs/chaos_experiments/stresschaos_experiment#usage) with the few required adjustments for my cluster.



***Experiment that Crashes the Dashboard***
```yaml
apiVersion: chaos-mesh.org/v1alpha1
kind: StressChaos
metadata:
  name: burn-cpu
  namespace: chaos-mesh
spec:
  mode: one
  selector:
    namespaces:
      - confluentplatform
  stressors:
    cpu:
      workers: 1
  duration: "30s"
  scheduler:
    cron: "@every 2m"
```

If you require any additional information please let me know.

--- a/api/v1alpha1/zz_generated.chaosmesh.go
+++ b/api/v1alpha1/zz_generated.chaosmesh.go
@@ -103,7 +103,7 @@ func (in *DNSChaos) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {
@@ -220,7 +220,7 @@ func (in *HTTPChaos) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {
@@ -337,7 +337,7 @@ func (in *IoChaos) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {
@@ -454,7 +454,7 @@ func (in *KernelChaos) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {
@@ -571,7 +571,7 @@ func (in *NetworkChaos) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {
@@ -688,7 +688,7 @@ func (in *PodChaos) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {
@@ -805,7 +805,7 @@ func (in *StressChaos) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {
@@ -922,7 +922,7 @@ func (in *TimeChaos) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {

--- a/cmd/chaos-builder/impl.go
+++ b/cmd/chaos-builder/impl.go
@@ -101,7 +101,7 @@ func (in *{{.Type}}) GetChaos() *ChaosInstance {
 	}
 
 	action := reflect.ValueOf(in).Elem().FieldByName("Spec").FieldByName("Action")
-	if !action.IsZero() {
+	if action.IsValid() {
 		instance.Action = action.String()
 	}
 	if in.Spec.Duration != nil {

